### PR TITLE
Inc 18/icons

### PR DESCRIPTION
**Pull request by Viarcinski**
add reusable icons componet to ui-kit

### CODE REVIEW

**Review by irinrudko — APPROVED** [review]
ok

**Review by maskpa100 — COMMENTED** [review]
Ok

File: @ui/ui-kit/Icon/IconsComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import MessageOutline from '@/public/sidebar-icons/message-circle-outline.svg'
+import Message from '@/public/sidebar-icons/message-circle.svg'
+import SearchOutline from '@/public/sidebar-icons/search-outline.svg'
+import Search from '@/public/sidebar-icons/search.svg'
+import TrendingUpOutline from '@/public/sidebar-icons/trending-up-outline.svg'
+import TrendingUp from '@/public/sidebar-icons/trending-up.svg'
+import BookmarkOutline from '@/public/sidebar-icons/bookmark-outline.svg'
+import Bookmark from '@/public/sidebar-icons/bookmark.svg'
+import LogoutOutline from '@/public/sidebar-icons/log-out-outline.svg'
+import Logout from '@/public/sidebar-icons/log-out.svg'
```
Comment (TURP1) [inline @ui/ui-kit/Icon/IconsComponent.tsx:18]: it's better to create an index file in the folder where all the svgs are located, and access them from here with the file index

File: components/dev/UiKit/Icons.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+const IconsDev = () => {
+  return (
+    <PageWrapper>
+      <div className={style.kitContainer}>
+
+        <div style={{ display: 'flex', textAlign: 'center', color: 'white' }}>
+        <div>
+          <h2>Default</h2>
+          <Icons.Home className={style.SideBar__Item} />
```
Comment (TURP1) [inline components/dev/UiKit/Icons.tsx:19]: it's inconvenient to specify a classname each time when creating an icon

File: components/dev/UiKit/Icons.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          <Icons.Create className={style.SideBar__Item} />
+          <Icons.Profile className={style.SideBar__Item} />
+          <Icons.Messenger className={style.SideBar__Item} />
+          <Icons.Search className={style.SideBar__Item} />
+          <Icons.Statistics className={style.SideBar__Item} />
+          <Icons.Logout className={style.SideBar__Item} />
+        </div>
+        <div>
+          <h2>Active</h2>
+          <Icons.Home className={style.SideBar__Item} isActive />
```
Comment (TURP1) [inline components/dev/UiKit/Icons.tsx:29]: isActive work`s incorrect. We don`t need to create such prop. CSS will do it for us.

File: components/dev/UiKit/Icons.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          <Icons.Create className={style.SideBar__Item} />
+          <Icons.Profile className={style.SideBar__Item} />
+          <Icons.Messenger className={style.SideBar__Item} />
+          <Icons.Search className={style.SideBar__Item} />
+          <Icons.Statistics className={style.SideBar__Item} />
+          <Icons.Logout className={style.SideBar__Item} />
+        </div>
+        <div>
+          <h2>Disabled</h2>
+          <Icons.Home className={style.SideBar__Item} isDisabled />
```
Comment (TURP1) [inline components/dev/UiKit/Icons.tsx:39]: same with isDisabled

**Review by TURP1 — CHANGES_REQUESTED** [review]
take a closer look at the work :active , pay attention to the padding

File: @ui/ui-kit/Icon/IconsComponent.tsx (hunk 1 above)
Comment (Viarcinski) [reply @ui/ui-kit/Icon/IconsComponent.tsx]: done

File: components/dev/UiKit/Icons.tsx (hunk 2 above)
Comment (Viarcinski) [reply components/dev/UiKit/Icons.tsx]: done

**Review by ilich01 — APPROVED** [review]

File: @ui/ui-kit/Icon/MakeIcon.tsx [hunk 5]
```diff
@@ -2,14 +2,15 @@ import { FC } from 'react'
 import { MakeIconPropsType } from './iconsType'
 import style from './IconsComponent.module.scss'
 import Link from 'next/link'
+import classNames from 'classnames'
 
 export const MakeIcon: FC<MakeIconPropsType> = (props) => {
-  const { OutlineIcon, Icon, className, text, isActive, isDisabled, Url } = props
+  const { OutlineIcon, Icon, customClass, text, isActive, isDisabled, url } = props
+  const className = classNames(style.IconWrapper, customClass, {[style.IconWrapperActive]: isActive, [style.IconWrapperDisabled]: isDisabled })
```
Comment (TURP1) [inline @ui/ui-kit/Icon/MakeIcon.tsx:9]: what is customClass for?

Comment (TURP1) [reply @ui/ui-kit/Icon/MakeIcon.tsx:9]: remove this line to the line below
 {[style.IconWrapperActive]: isActive

**Review by maskpa100 — COMMENTED** [review]
ok

**Review by maskpa100 — COMMENTED** [review]
ok

**Pull request merged**